pkgs/development/python-modules/colorcet/default.nix

```diff
@@ -34,9 +34,9 @@ buildPythonPackage rec {
     export HOME=$(mktemp -d)
     mkdir -p $HOME/.config/matplotlib
     echo "backend: ps" > $HOME/.config/matplotlib/matplotlibrc
+    ln -s $HOME/.config/matplotlib $HOME/.matplotlib
```

Comment:
nice catch! can we use just use `$HOME/.matplotlib`? i.e `echo "backend: ps" > $HOME/.matplotlib/matplotlibrc`

Reply:
Mmmmno, it doesn't appear to pay any attention to that.

Reply:
oh, no problem. Thanks for catching this